sockeye/output_handler.py

```diff
@@ -48,11 +50,15 @@ class OutputHandler:
     Abstract output handler interface
     """
 
-    def handle(self, t_input: sockeye.inference.TranslatorInput, t_output: sockeye.inference.TranslatorOutput):
+    def handle(self,
```

Comment:
The idea is that we want to record as much of the total translation time as possible, so time() is the last thing we call before translate() and the first thing we call after.  To move timing inside TranslatorOutput, we could add time() calls a few methods down in the inference code (but not capture absolutely everything) or split timing across methods by tracking start and end times.  I think under the circumstances keeping timing outside is the simplest and most accurate.